.github/workflows/CI.yml

```diff
@@ -13,16 +13,34 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        version:
-          - '1.6'     # LTS
-          - '1'       # current stable
-          - 'nightly'
-        os:
-          - ubuntu-latest
-          - macOS-latest
-          - windows-latest
-        arch:
-          - x64
+        include:
+          - version: '1.6' # old LTS
+            os: ubuntu-latest
+            arch: x64
+          - version: '1.10' # current stable
+            os: ubuntu-latest
+            arch: x64
+          - version: '~1.11.0-0' # next release
+            os: ubuntu-latest
+            arch: x64
+          - version: '1.6' # old LTS
+            os: ubuntu-latest
+            arch: x64
+          - version: 'nightly' # dev
+            os: ubuntu-latest
+            arch: x64
+          #- version: '1' # x86 ubuntu -- disabled since PyCall/conda is broken on this platform
+          #  os: ubuntu-latest
+          #  arch: x86
+          - version: '1' # x86 windows
+            os: windows-latest
+            arch: x86
+          - version: '1' # x64 windows
+            os: windows-latest
+            arch: x64
+          - version: '1' # x64 macOS
+            os: macos-latest
+            arch: x64
```

Comment:
Keeping this list updated and maintained seems to be a real hassle. Can we go back to how things were before with using `matrix`? Adding `1.11-nightly` just for testing the next release temporarily makes sense to me. And if we need to disable certain platforms, we could use:
```yml
       exclude:
         - version: '1' # x86 ubuntu -- disable since PyCall/conda is broken on this platform
           os: ubuntu-latest
           arch: x86
```
should do the trick.

Reply:
The only part of the include list to update is the number for latest stable and next release, which is the same regardless of whether it is a matrix or list. I like a list more, since we can set the list of platforms that are actually interesting sparsely selecting interesting variations on the matrix, rather than running all versions for all platforms for all OS

Reply:
It is basically a build "T"ee instead of a Matrix. One platform runs tests in depth (ubuntu x64) and other platforms run the tests in breadth ("1.x")